ddtrace/internal/_encoding.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Any
+from typing import Union
+
+
+class MsgpackEncoder(object):
+    content_type: str
+
+    def _decode(self, data: Union[str, bytes]) -> Any: ...
```

Comment:
⚠️  missing newline

Reply:
Ahh thanks for the catch! I forgot to run black before creating this PR...